design-reference/docs/azure-simple.md

```diff
@@ -0,0 +1,206 @@
+| appgateway\_http\_listener\_protocol | The Http Listener protocol for the Appication Gateway to be created | string | `"Https"` | no |
+| docker\_registry\_server\_password | The password used to authenticate with the container regitry | string | `""` | no |
+| docker\_registry\_server\_url | The url of the container registry that will be utilized to pull container into the Web Apps for containers | string | `"index.docker.io"` | no |
+| docker\_registry\_server\_username | The username used to authenticate with the container registry | string | `""` | no |
+| name | The name of the deployment.  This will be used across the resource created in this solution | string | n/a | yes |
+| resource\_group\_location | The deployment location of resource group container all the resources | string | n/a | yes |
+| subnet\_names | A list of public subnets inside the vNet. | list | `<list>` | no |
+| subnet\_prefixes | The address prefix to use for the subnet. | list | `<list>` | no |
+| subnet\_service\_endpoints | The list of service endpoints that will be given to each subnet | list | `<list>` | no ||
+
+
+### Module configuration
+
+<!---
+This content was generated automatically from `main.tf`.
+--->
+
+Each module in the `azure-simple` template contains its own configuration options. Click the links below to learn more.
+ - [`service-plan`](https://microsoft.github.io/orion/service-plan#Configuration)
+ - [`app-service`](https://microsoft.github.io/orion/app-service#Configuration)
+ - [`app-insights`](https://microsoft.github.io/orion/app-insights#Configuration)
+
+
+### Deployment steps
+
+Create a new Terraform template directory and add a `main.tf` file. Here's a sample that uses the `azure-simple` template.
+
+```
+Add sample here. Required.
+```
+
+Call the `terraform init`, `terraform plan`, and `terraform apply` commands within the template directory to initialize the Terraform deployment then write and apply the plan.
+
+```
+Add code block here. Required.
+```
+
+<!---
+This content will be generated by [TBD].
+--->
+
+The following resources will be created:
+
+ - Linux App Service Plan
+ - App Service Container + staging slot
+ - Application Insights Instance
+ - vnet + subnet for app gateway
+ - Application Gateway with an enabled Firewall
+ - Traffic Manager Profile
+ - Traffic Manager Endpoint
+ - Azure Monitoring Rules
+ - Public IP(for TM + AG)
+
+
+### Outputs
+
+<!---
+This content was generated automatically from `ouputs.tf` using `terraform-docs` (https://github.com/segmentio/terraform-docs).
+--->
+
+| Name | Description |
+|------|-------------|
+| app\_service\_default\_hostname |  |
+| app\_service\_name |  
+
+
+## Features
+
+<!---
+This content will be generated by [TBD].
+--->
+
+| Feature            | Description                        |
+|--------------------|------------------------------------|
+| Performance |  |
+| Scaling |  |
+| Cost | |
+| Security | |
+| Authentication | |
```

Comment:
I'd provide a example saying that we're using active directory for identity based authentication